examples/event-webhooks-local-flask/local_app/benchling_app/handler.py

```diff
@@ -0,0 +1,54 @@
+from typing import Any
+
+from benchling_api_client.webhooks.v0.beta.models.event_created_webhook_v0_beta import (
+    EventCreatedWebhookV0Beta,
+)
+from benchling_api_client.webhooks.v0.beta.models.v2_entity_registered_event import (
+    V2EntityRegisteredEvent as V2EntityRegisteredEventBeta,
+)
+from benchling_api_client.webhooks.v0.beta.models.webhook_envelope import (
+    WebhookEnvelope as WebhookEnvelopeV0Beta,
+)
+from benchling_sdk.apps.framework import App
+
+from local_app.benchling_app.setup import init_app_from_webhook
+from local_app.benchling_app.sync_data import sync_event_data
+from local_app.lib.logger import get_logger
+
+logger = get_logger()
+
+
+class UnsupportedWebhookError(Exception):
+    pass
+
+
+def handle_webhook(webhook_dict: dict[str, Any]) -> None:
+    logger.debug("Handling webhook with payload: %s", webhook_dict)
+    webhook = WebhookEnvelopeV0Beta.from_dict(webhook_dict)
+    app = init_app_from_webhook(webhook)
+    # Could also choose to route on webhook.message.type
+    if isinstance(webhook.message, EventCreatedWebhookV0Beta):
+        # Since we'll be receiving ALL entity registrations, add a filter to only
+        # work with events that meet our criteria (e.g., specific schema)
+        if _is_target_event(app, webhook.message):
```

Comment:
@edwin-benchling This filtering logic is one of the main differences from the other example App that would merit closer review.